### ISSUE TITLE
mpii dataset json file

### ISSUE THREAD

**Issue by txytju** [ilovepose/fast-human-pose-estimation.pytorch#3]
Thanks for your great work!
When I tried to run the training on mpii dataset,  mpii/anno.json is required. However, original mpii dataset annotation is in mat form. So where can I find this json file?

**Comment by djangogo** [comment]
Hi,
You can download it from:
https://1drv.ms/f/s!AhIXJn_J-blW00SqrairNetmeVu4
or
https://drive.google.com/drive/folders/1En_VqmStnsXMdldXA6qpqEyDQulnmS3a?usp=sharing

**Comment by HuAndrew** [comment]
**For MPII data**, please download from [MPII Human Pose Dataset](http://human-pose.mpi-inf.mpg.de/). The original annotation files are in matlab format. We have converted them into json format, you also need to download them from [OneDrive](https://1drv.ms/f/s!AhIXJn_J-blW00SqrairNetmeVu4) or [GoogleDrive](https://drive.google.com/drive/folders/1En_VqmStnsXMdldXA6qpqEyDQulnmS3a?usp=sharing).
Extract them under {POSE_ROOT}/data, and make them look like this:
```
${POSE_ROOT}
|-- data
`-- |-- mpii
    `-- |-- annot
        |   |-- gt_valid.mat
        |   |-- test.json
        |   |-- train.json
        |   |-- trainval.json
        |   `-- valid.json
        `-- images
            |-- 000001163.jpg
            |-- 000003072.jpg
```

**Comment by 738654805** [comment]
@HuAndrew 老哥这个test.json只有7000+，是正常么？还是需要转化一下才能得到11000+的？

**Comment by HuAndrew** [comment]
The all data is provided by MSRA, please see [HRNet official Project.](https://github.com/leoxiaobin/deep-high-resolution-net.pytorch)

**Comment by MingSun-Tse** [comment]
> @HuAndrew 老哥这个test.json只有7000+，是正常么？还是需要转化一下才能得到11000+的？

Hi buddy, did you figure out why there are only 7000+ samples in the test set? Thanks!

**Comment by xizero00** [comment]
> > @HuAndrew 老哥这个test.json只有7000+，是正常么？还是需要转化一下才能得到11000+的？
> 
> Hi buddy, did you figure out why there are only 7000+ samples in the test set? Thanks!

7000+ samples are OK.
Some samples are ignored because they DO NOT provide center position.
